modules/modules.nix

```diff
@@ -155,6 +155,7 @@ let
     (loadModule ./services/taskwarrior-sync.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/udiskie.nix { })
     (loadModule ./services/unclutter.nix { })
+    (loadModule ./services/keynav.nix { })
```

Comment:
Since the module uses systemd it needs to be Linux only:
```suggestion
    (loadModule ./services/keynav.nix { condition = hostPlatform.isLinux; })
```